scipy/stats/__init__.py

```diff
@@ -435,6 +436,15 @@
    PearsonRNearConstantInputWarning
    SpearmanRConstantInputWarning
 
+
+Classes returned by some functions
+==================================
+
+.. autosummary::
+   :toctree: generated/
+
+   scipy.stats._result_classes.OddsRatioResult
```

Comment:
The plan is to remove this and get the documentation for `OddsRatioResult` to render otherwise, right?

Reply:
That is the goal, but so far I haven't been able to get the online docs for `OddsRatioResult` to render without having something like this in the `stats` module docstring in `__init__.py`.

Reply:
Unresolving this based on the progress in gh-13048

Reply:
The documentation for `OddsRatioResult` is now generated the same as the `RelativeRiskResult` and `BinomTestResult`.